### PR TITLE
Service Interconnect Onboarding PR

### PR DESCRIPTION
**Pull request by smatula**

### CODE REVIEW

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-rhpit-interop-tests-main-smatula-ocp4.14-lp-interop-si-interop-aws-smatula

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-rhpit-interop-tests-main-smatula-ocp4.14-lp-interop-si-interop-aws-smatula

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-rhpit-interop-tests-main-smatula-ocp4.14-lp-interop-si-interop-aws-smatula

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-rhpit-interop-tests-main-smatula-ocp4.14-lp-interop-si-interop-aws-smatula

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-rhpit-interop-tests-main-smatula-ocp4.13-lp-interop-si-interop-aws-smatula

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-rhpit-interop-tests-main-smatula-ocp4.13-lp-interop-si-interop-aws-smatula

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-rhpit-interop-tests-main-smatula-ocp4.14-lp-interop-si-interop-aws-smatula

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-rhpit-interop-tests-main-smatula-ocp4.13-lp-interop-si-interop-aws-smatula

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-rhpit-interop-tests-main-smatula-ocp4.13-lp-interop-si-interop-aws-smatula

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-rhpit-interop-tests-main-smatula-ocp4.13-lp-interop-si-interop-aws-smatula

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-rhpit-interop-tests-main-smatula-ocp4.13-lp-interop-si-interop-aws-smatula

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-rhpit-interop-tests-main-smatula-ocp4.13-lp-interop-si-interop-aws-smatula

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-rhpit-interop-tests-main-service-interconnect-ocp4.13-lp-interop-rhsi-interop-aws

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-rhpit-interop-tests-main-service-interconnect-ocp4.13-lp-interop-rhsi-interop-aws

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-rhpit-interop-tests-main-service-interconnect-ocp4.13-lp-interop-rhsi-interop-aws

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-rhpit-interop-tests-main-service-interconnect-ocp4.14-lp-interop-rhsi-interop-aws

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-rhpit-interop-tests-main-service-interconnect-ocp4.13-lp-interop-rhsi-interop-aws

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-rhpit-interop-tests-main-service-interconnect-ocp4.13-lp-interop-rhsi-interop-aws

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-rhpit-interop-tests-main-service-interconnect-ocp4.13-lp-interop-rhsi-interop-aws

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-rhpit-interop-tests-main-service-interconnect-ocp4.13-lp-interop-rhsi-interop-aws

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-rhpit-interop-tests-main-service-interconnect-ocp4.14-lp-interop-rhsi-interop-aws

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-rhpit-interop-tests-main-service-interconnect-ocp4.14-lp-interop-rhsi-interop-aws

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-rhpit-interop-tests-main-service-interconnect-ocp4.14-lp-interop-rhsi-interop-aws

File: ci-operator/step-registry/rhsi/run-tests/rhsi-run-tests-commands.sh [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+./run-test.sh || status="$?" || :
+
+mkdir -p $ARTIFACT_DIR/result
+
+# Copy results to ARTIFACT_DIR
+cp -r /result/* $ARTIFACT_DIR/result 2>/dev/null || :
+cp -r /tmp/test.out $ARTIFACT_DIR/result 2>/dev/null || :
+
+# Prepend junit_ to result xml files
+rename '/junit' '/junit_rhsi' ${ARTIFACT_DIR}/result/*.xml 2>/dev/null || :
```
Comment (chetna14manku) [inline ci-operator/step-registry/rhsi/run-tests/rhsi-run-tests-commands.sh:24]: @smatula I see xmls are not getting renamed [here](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/36424/rehearse-36424-periodic-ci-skupperproject-skupper-openshift-smoke-test-image-main-service-interconnect-ocp4.14-lp-interop-rhsi-interop-aws/1706360517720805376/artifacts/rhsi-interop-aws/rhsi-run-tests/artifacts/result/) can you recheck

Comment (smatula) [reply ci-operator/step-registry/rhsi/run-tests/rhsi-run-tests-commands.sh]: @chetna14manku Updated code. rename did not exist on container. Filenames  are now getting updated.

**Pull request merged**